eosq/src/i18n/zh/footer.i18n.ts

```diff
@@ -1,15 +1,16 @@
 export default {
   footer: {
-    telegram: "Telegram技术支持群",
-    subTitle: "由 dfuse 为 EOS 社区而开发",
-    copyright: "© {{year}} dfuse Platform Inc. 版权所有	",
-    madeWithLove: "Made with Love by",
+    telegram: "Telegram 社区&团队支持群",
+    subTitle: "need to translate",
+    aboutDfuse: "dfuse 是一个开源区块链数据平台",
+    copyright: "© 2018-{{year}} dfuse Platform Inc. 版权所有",
     privacyPolicy: "隐私政策",
-    privacyPolicyLink: "https://www.eoscanada.com/en/privacy-policy",
+    privacyPolicyLink: "https://dfuse.io/zh/%E9%9A%90%E7%A7%81%E6%94%BF%E7%AD%96/",
     termsOfServices: "服务条款",
-    termsOfServicesLink: "https://www.dfuse.io/en/terms",
-    eoscanada: "EOS Canada",
-    eoscanadaLink: "https://www.eoscanada.com",
-    eoscanadaLinkCombined: "<0>EOS Canada</0> 用爱制造"
+    termsOfServicesLink: "https://dfuse.io/en/terms"
   }
 }
+
+/**
```

Comment:
Weird, what's that?

Reply:
Oh it's the comment that should go on `subTitle` not translated. We could remove the `subTitle` key then and it's `en` counterpart.

Reply:
agreed